3.32 is still compatible with other 3.14-API editors

--- origsrc/gtkhtml-3.32.1/components/editor/Makefile.am	2010-09-22 00:14:26.000000000 -0500
+++ src/gtkhtml-3.32.1/components/editor/Makefile.am	2010-11-15 16:31:10.807818700 -0600
@@ -71,19 +71,19 @@ libgtkhtml_editor_4_0_la_LIBADD = \
 	$(top_builddir)/gtkhtml/libgtkhtml-@EDITOR_API_VERSION@.la
 libgtkhtml_editor_4_0_la_LDFLAGS = -no-undefined
 else
-lib_LTLIBRARIES = libgtkhtml-editor-3.14.la
-libgtkhtml_editor_3_14_la_SOURCES = $(csources) 
-libgtkhtml_editor_3_14_la_LIBADD = \
+lib_LTLIBRARIES = libgtkhtml-editor.la
+libgtkhtml_editor_la_SOURCES = $(csources) 
+libgtkhtml_editor_la_LIBADD = \
 	@GTKHTML_LIBS@				\
 	$(top_builddir)/gtkhtml/libgtkhtml-@EDITOR_API_VERSION@.la
-libgtkhtml_editor_3_14_la_LDFLAGS = -no-undefined
+libgtkhtml_editor_la_LDFLAGS = -no-undefined
 endif
 
 gtkhtml_editor_test_SOURCES = main.c
 
 gtkhtml_editor_test_LDADD = \
 	$(GTHREAD_LIBS)				\
-	libgtkhtml-editor-@EDITOR_API_VERSION@.la			\
+	libgtkhtml-editor.la			\
 	$(top_builddir)/gtkhtml/libgtkhtml-@EDITOR_API_VERSION@.la
 
 gtkhtml-editor-marshal.h: gtkhtml-editor-marshal.list $(GLIB_GENMARSHAL)
--- origsrc/gtkhtml-3.32.1/components/editor/gtkhtml-editor.pc.in	2010-09-22 00:14:26.000000000 -0500
+++ src/gtkhtml-3.32.1/components/editor/gtkhtml-editor.pc.in	2010-12-16 02:49:52.726756100 -0600
@@ -9,5 +9,5 @@ Name: gtkhtml-editor
 Description: HTML editor widget
 Requires: libgtkhtml-@GTKHTML_API_VERSION@
 Version: @VERSION@
-Libs: -L${libdir} -lgtkhtml-editor-@GTKHTML_API_VERSION@
+Libs: -L${libdir} -lgtkhtml-editor
 Cflags: -I${includedir}/libgtkhtml-@GTKHTML_API_VERSION@/editor
