--- origsrc/gtkhtml-3.18.3/components/html-editor/Makefile.am	2008-04-04 04:06:35.000000000 -0500
+++ src/gtkhtml-3.18.3/components/html-editor/Makefile.am	2008-07-10 00:46:34.750000000 -0500
@@ -140,11 +140,11 @@
 if EDITOR_SHLIB
 MY_PRGS =
 MY_LIBS = libgnome-gtkhtml-editor-3.14.la
-location = "${privlibdir_in_server_file}/libgnome-gtkhtml-editor-$(EDITOR_API_VERSION)$(SOEXT)"
+location = "${privlibdir_in_server_file}/$(SOPREFIX)gnome-gtkhtml-editor-$(EDITOR_API_VERSION)"
 else
 MY_PRGS = gnome-gtkhtml-editor-3.14
 MY_LIBS =
-location = "${libexecdir}/$(MY_PRGS)$(EXEEXT)"
+location = "${libexecdir}/$(MY_PRGS)"
 endif
 
 libgnome_gtkhtml_editor_3_14_la_SOURCES = \
--- origsrc/gtkhtml-3.18.3/configure.in	2008-06-29 00:40:02.000000000 -0500
+++ src/gtkhtml-3.18.3/configure.in	2008-07-10 00:46:35.234375000 -0500
@@ -57,15 +57,19 @@
 case "$host" in
 *-mingw*)
     os_win32=yes
-    SOEXT='.dll'
+    SOPREFIX='lib'
+    ;;
+*-cygwin*)
+    os_win32=no
+    SOPREFIX='cyg'
     ;;
 *)  os_win32=no
-    SOEXT='.so'
+    SOPREFIX='lib'
     ;;
 esac
 AC_MSG_RESULT([$os_win32])
 AM_CONDITIONAL(OS_WIN32, [test $os_win32 = yes])
-AC_SUBST(SOEXT)
+AC_SUBST(SOPREFIX)
 
 GNOME_COMPILE_WARNINGS(yes)
 CFLAGS="$CFLAGS $warning_flags"
